sandbox/modules/identity/src/main/java/org/opensearch/identity/User.java

```diff
@@ -51,6 +53,9 @@ public User(final String username, final String bcryptHash, Map<String, String>
     @JsonProperty(value = "attributes")
     private Map<String, String> attributes = Collections.emptyMap();
 
+    @JsonProperty(value = "permissions")
+    private List<String> permissions = Collections.emptyList();
```

Comment:
Not blocking this change: I think we should retrieve permissions from the store that @scrawfor99 has created

Reply:
Agreed. This is added here to show a permissions check working end to end and the magic of permissions resolution (Subject -> List of Permissions) happens in `doGetAuthorizationInfo` of the `InternalRealm`. That method can be updated when the long-term permissions store is decided upon and implemented.